vmaas_sync/pkg_sync.go

```diff
@@ -111,15 +130,30 @@ func storePackageDetails(tx *gorm.DB, advisoryIDs map[utils.Nevra]int, nameIDs m
 			utils.Log("nevra", nevra.String()).Warn("Did not find matching advisories for nevra")
 			continue
 		}
-		toStore = append(toStore, models.Package{
+
+		next := models.Package{
 			NameID:          nameIDs[nevra.Name],
 			EVRA:            nevra.EVRAString(),
 			DescriptionHash: desc[:],
 			SummaryHash:     sum[:],
 			AdvisoryID:      advisoryIDs[nevra],
-		})
+		}
+
+		for _, op := range oldPackages {
```

Comment:
seems to be a bit exhausting, right? What to load old packages to map[NameID-Evra]models.Package and then use it for old package - new one associations?